java/flight/flight-sql-jdbc-core/src/test/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcDriverTest.java

```diff
@@ -142,15 +142,15 @@ public void testConnectWithInsensitiveCasePropertyKeys() throws Exception {
              driver.connect("jdbc:arrow-flight://" +
                      dataSource.getConfig().getHost() + ":" +
                      dataSource.getConfig().getPort() + "?" +
-                     "UseEncryptiOn=false",
+                     "UseEncryptIon=false",
```

Comment:
shouldn't it be?

```suggestion
                     "UseEncryption=false",
```

Reply:
That is covered by the normal test flow:
https://github.com/apache/arrow/blob/16fd249385aba5a1c80708de894185cebd601cad/java/flight/flight-sql-jdbc-core/src/test/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcDriverTest.java#L115-L122

These tests, otoh are for nonstandard forms:
https://github.com/apache/arrow/blob/16fd249385aba5a1c80708de894185cebd601cad/java/flight/flight-sql-jdbc-core/src/test/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcDriverTest.java#L138-L145

I've selected `Encrypt` + `Ion` because both are words while still honoring the requirement that it be a non-standard case of `Encryption`.